tensorflow/lite/micro/kernels/lstm_eval_test.h

```diff
@@ -319,6 +322,14 @@ OpDataLSTM CreateLstmOpDataFloat(
 }
 
 /*Test Functions Below Here*/
+template <typename T>
```

Comment:
not sure that this change needs to be part of the current PR, but let's go for it since there are a lot of moving pieces.